examples/pytorch/model_zoo/chem/property_prediction/regression.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding:utf-8 -*-
+"""Sample training code
+"""
+
+import argparse
+import torch as th
+import torch.nn as nn
+from dgl.data import alchemy
```

Comment:
If you make modifications as I suggested in `python/dgl/data/__init__.py`, you can now directly `from dgl.data import TencentAlchemyDataset`